data/README.md

```diff
@@ -0,0 +1,43 @@
+# Data sources
+
+## data/target_scan_8 (downloaded on 1.6.2024)
```

Comment:
I think you are missing name od file here. And I don't see this file in the repo. There is only `Gene_info.txt` and that is described in a different part.

Reply:
I see the confusion. This line is supposed to inform about the whole folder. The specific files are then described under this title. However, it is a confusing structure, I will rewrite it based on your other [comment](https://github.com/ML-Bioinfo-CEITEC/miRNA/pull/6#discussion_r1692777983)